presto-hive/src/main/java/com/facebook/presto/hive/PrestoS3FileSystemStats.java

```diff
@@ -247,4 +258,9 @@ public void updateAwsThrottleExceptionsCount(long throttleExceptionsCount)
     {
         awsThrottleExceptions.update(throttleExceptionsCount);
     }
+
+    public void addAwsRequestLatency(double latency)
```

Comment:
Make this method take a `Duration`, as this prevents callers from mixing up units